tools/cli/build.sh

```diff
@@ -98,8 +98,15 @@ create_cli_packages() {
   echo $dirIndex > ./build/content.json
 };
 
-platforms="mac linux windows"
-archs="386 amd64"
+
+if [ "$LOCAL_BUILD" = "true" ]; then \
+  platforms="$LOCAL_OS" && \
+  archs="$LOCAL_ARCH"; \
+else \
+  platforms="mac linux windows" && \
+  archs="386 amd64"; \
+fi && \
```

Comment:
Why make the distinction here? Why don't we just pass `platforms` and `archs` through from the gradle project. `crosscompile` then means: pass this whole list. Thoughts?